lib/active_admin/exception_app.rb

```diff
@@ -0,0 +1,79 @@
+require 'active_admin/view_helpers'
+
+module ActiveAdmin
+  class ExceptionController < ActionController::Base
+    include Rails.application.routes.url_helpers
+    helper ::ActiveAdmin::ViewHelpers
+
+    def self.call(env)
+      response = action(:index)
+      response.call(env)
+    end
+
+    def index
+    end
+
+
+    def active_admin_application
+      ActiveAdmin.application
+    end
+
+    helper_method :active_admin_application
+
+    def active_admin_namespace
+      namespace = active_admin_application.default_namespace
+      active_admin_application.namespace(namespace)
+    end
+
+    helper_method :active_admin_namespace
+
+
+    def active_admin_config
+      ::ActiveAdmin::Page.new(active_admin_namespace, "Error", {})
+    end
+
+    helper_method :active_admin_config
+
+    def current_menu
+      active_admin_config.navigation_menu
+    end
+
+    helper_method :current_menu
+
+    def env
+      request.env
+    end
+
+    helper_method :env
+
+    def authorized?(action, subject = nil)
```

Comment:
Unused method argument - `action`. If it's necessary, use `_` or `_action` as an argument name to indicate that it won't be used. You can also write as `authorized?(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `subject`. If it's necessary, use `_` or `_subject` as an argument name to indicate that it won't be used. You can also write as `authorized?(*)` if you want the method to accept any arguments but don't care about them.